aiohttp/client_utils.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+import attr
+
+
+@attr.s(frozen=True, slots=True)
+class ClientTimeout:
+    total = attr.ib(type=Optional[float], default=None)
+    connect = attr.ib(type=Optional[float], default=None)
+    sock_read = attr.ib(type=Optional[float], default=None)
+    sock_connect = attr.ib(type=Optional[float], default=None)
+    sock_close = attr.ib(type=Optional[float], default=None)
```

Comment:
Please use different timeout classes for HTTP requests and WebSockets.

`ClientWSTimeout` can be *derived* from `ClientTimeout` but `ClientTimeout` should be untouched.
`sock_close` is a bad name, `ws_close` is better. Later something like `ws_receive` and `ws_send` should be added.